lms-canvas/src/main/java/no/sikt/nva/lms/CanvasLaunchHandler.java

```diff
@@ -0,0 +1,47 @@
+package no.sikt.nva.lms;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.util.Collections;
+import java.util.Set;
+import java.util.concurrent.ConcurrentSkipListSet;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import sikt.lti.tp.aws.ApiGatewayLambdaLaunchHandler;
+
+public class CanvasLaunchHandler extends ApiGatewayHandler<Void, String> {
+
+    private static final URI DLR_BASE_URL = URI.create("https://dlr.unit.no");
```

Comment:
DLR_BASE_URL and API_HOST will differ in different environments. Should be read from environment.